### PR TITLE
First version

### PR DESCRIPTION
**Pull request by viktorklochkov**

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -157,3 +157,14 @@ def preds_prob(df, preds, true, dataset, output_path):
     plt.show()
     fig.tight_layout()
     fig.savefig(str(output_path)+'/test_best_pred.png')
+
+
+def diff_SB(df, signal_label):
+    dfs = df[df[signal_label]==1]
+    dfb = df[df[signal_label]==0]
```
Comment (shahidzk1) [inline helper.py:179]: I would try something like 
`
dfb = df[df[signal_label]==0]
if len(df[signal_label].unique()) < 3:
      dfs = df[df[signal_label]==1]
else:
    dfsp = df[df[signal_label]==1]

  dfss = df[df[signal_label]>1]
 `

Comment (conformist89) [reply helper.py:179]: Could you please explain more explicitly these changes?

**Comment by viktorklochkov** [comment]
I think we also need a proper README, with input file example and how you expect people to interact with this code

**Comment by conformist89** [comment]
> I think we also need a proper README, with input file example and how you expect people to interact with this code

For sure It will be implemented as soon I fix TrainTestXGBoost class. I have such an example but now it's included to gitignore

**Pull request merged**